src/app/docs/waterfall-indicator/waterfall-indicator-docs.component.html

```diff
@@ -0,0 +1,247 @@
+        </p>
+      </sky-docs-design-guideline>
+    </sky-docs-demo-page-section>
+    
+    <sky-docs-demo-page-section
+      heading="Content"
+    >
+      <sky-docs-design-guideline>
+        <p>
+          Use a waterfall title in place of an ordinary page title to provide context for the overall task. Begin the title with a verb to describe the action that the waterfall steps take when they are all completed, similar to the way modal titles are constructed.
+        </p>
+        <p>
+          Step labels in waterfall progress indicators start with a number and concisely describe the objective. Use the <b>&lt;verb> &lt;direct object></b> format. In the label for the last step, describe the cumulative change that occurs at the end of the process.
+        </p>
+        <p>
+          Step action (button) labels closely follow step labels and use the same <b>&lt;verb> &lt;direct object></b> format, but they do not need to be identical. For the first step action label, be generic and conversational (“Get started”), and for completed step actions, use verbs appropriate to post hoc actions (“Change”, “Edit”).
+        </p>
+      </sky-docs-design-guideline>
+
+      <sky-docs-design-guideline
+        heading="Layout"
+      >
+        <p>
+          Waterfall progress indicators are 400 pixels wide and centered inside the page. The waterfall title is the page title. Do not combine waterfall indicators with other content or actions. If waterfall indicators require extra context such as descriptions, links, or instructions, include it after the waterfall title and before the first step.
+        </p>
+
+        <sky-docs-design-guideline-thumbnail
+          caption="Do use waterfall progress indicators as the only content on a page while the user completes them."
+          captionType="success"
+          imageSource="~/assets/img/guidelines/passive-indicator/waterfall-progress-layout.png"
+        >
+        </sky-docs-design-guideline-thumbnail>
+      </sky-docs-design-guideline>
+    </sky-docs-demo-page-section>
+
+    <sky-docs-demo-page-section
+      heading="Related information"
+    >
+      <sky-docs-design-guideline
+        heading="Components"
+      >
+        <ul>
+          <li><a href="https://developer.blackbaud.com/skyux/components/modal">Modal</a></li>
+          <li><a href="https://developer.blackbaud.com/skyux/components/progress-indicator/passive-progress-indicator">Passive progress indicator</a></li>
+          <li><a href="https://developer.blackbaud.com/skyux/components/wizard">Wizard</a></li>
+        </ul>
+      </sky-docs-design-guideline>
+
+      <sky-docs-design-guideline
+        heading="Guidelines"
+      >
+        <ul>
+          <li><a href="https://developer.blackbaud.com/skyux/design/guidelines/form-design">Form design</a></li>
+        </ul>
+      </sky-docs-design-guideline>
+    </sky-docs-demo-page-section>
+
+  </sky-docs-design-guidelines>
```

Comment:
We need to add the JSDoc comments to the components and copy over the existing content on the website (minus `This property accepts...`). I added more detail about this on the toggle switch and key info PRs, but let me know those aren't clear.

Reply:
Done! Although, it's worth noting that some fields are still empty because they weren't covered in the current docs, so I'll need a developer to check behind me.